DDDEastAnglia.Tests/SponsorModelQueryBuilder.cs

```diff
@@ -0,0 +1,15 @@
+using DDDEastAnglia.Controllers;
+using DDDEastAnglia.DataAccess;
+
+namespace DDDEastAnglia.Tests
+{
+    public class SponsorModelQueryBuilder
```

Comment:
Again, is this buying us much or could we make do with a simpler SUT Factory for now?